src/core/lib/surface/call.cc

```diff
@@ -392,13 +388,13 @@ class FilterStackCall final : public Call {
   /* Contexts for various subsystems (security, tracing, ...). */
   grpc_call_context_element context_[GRPC_CONTEXT_COUNT] = {};
 
-  ManualConstructor<SliceBufferByteStream> sending_stream_;
+  ManualConstructor<SliceBuffer> sending_stream_;
```

Comment:
Suggest calling these two fields `send_message_` and `recv_message_`, since they're no longer streams.